client/verta/tests/test_dataset_versioning/test_dataset_version.py

```diff
@@ -4,7 +4,7 @@
 
 
 class TestDatasetVersion:
-    def test_create_get_path(self, client, created_datasets):
+    def test_create_get_path(self, client, created_datasets, capsys):
```

Comment:
Let's actually make these new, separate test functions. `capsys` prevents the entire test from outputting anything, which would make it harder to debug these existing tests.

Reply:
Done.